spark/v3.2/build.gradle

```diff
@@ -64,6 +66,8 @@ project(':iceberg-spark:iceberg-spark-3.2') {
     compileOnly("org.apache.spark:spark-hive_2.12:${sparkVersion}") {
       exclude group: 'org.apache.avro', module: 'avro'
       exclude group: 'org.apache.arrow'
+      // to make sure io.netty.buffer only comes from project(':iceberg-arrow')
+      exclude group: 'io.netty', module: 'netty-buffer'
```

Comment:
Do you need to add the same thing for the extensions?

https://github.com/apache/iceberg/blob/6c9b78b168ade472d5574a93c8bc7e3704f0675c/spark/v3.2/build.gradle#L136-L139

Reply:
+1. I think we should add the exclusion to all Spark imports right?

Reply:
Sorry @kbendick and @rdblue. Dig a bit deeper. The shaded netty buffer will always use the `forced` version by looking into the shaded class in the runtime jar file(iceberg-spark-3.2-runtime-0.13.0-SNAPSHOT.jar). In our case, it will use `4.1.68.Final`. This `exclude` doesn't take any effect. We have to think about other way to do that, like bump up netty buffer's version to 4.1.68.Final in iceberg-arrow. What do you think? cc @aokolnychyi

Reply:
We don't actually need this line make jmh work with the new change in iceberg-arrow, but I still keep them to make sure netty buffer only comes from iceberg-arrow. I'm open to any suggestion.